pyproject.toml

```diff
@@ -144,6 +144,7 @@ mongomock = "^3.18.0"
 black = "^19.10b0"
 flake8 = "^3.8.3"
 flake8-docstrings = "^1.5.0"
+pytype = "^2020.6.1"
```

Comment:
please remove this. We are using `mytype` on `master`

Reply:
Okay, exactly that was what I wasn't sure of. Should I then revert back changes made to `poetry.lock` as well because it seems like there isn't any dependency update to do if this goes away?

Reply:
sounds good 👍